hangman.py

```diff
@@ -113,21 +124,68 @@ def hangman(secret_word):
     
     Follows the other limitations detailed in the problem write-up.
     '''
-    # FILL IN YOUR CODE HERE AND DELETE "pass"
-    pass
-
+    print('''
+    __  __                                           ___ ____ 
+   / / / /___ _____  ____ _____ ___  ____ _____     <  // __ \\
+  / /_/ / __ `/ __ \/ __ `/ __ `__ \/ __ `/ __ \    / // / / /
+ / __  / /_/ / / / / /_/ / / / / / / /_/ / / / /   / // /_/ / 
+/_/ /_/\__,_/_/ /_/\__, /_/ /_/ /_/\__,_/_/ /_/   /_(_)____/  
+                  /____/                                      
+''')
+    # print(secret_word)
+    print(f'I am thinking of a word that is {len(set(secret_word))} letters long.')
+    guesses = 6
+    letters_guessed = []
+    vowels = ['a', 'e', 'i', 'o', 'u']
+    warnings = 3
+    while 1:
+        if is_word_guessed(secret_word, letters_guessed):
+            total_score = guesses * len(set(secret_word))
+            print(f'\033[7mCongratulations! You won, the secret word was - {secret_word}\033[0m')
+            print(f'\033[7mYour total score is {total_score}\033[0m')
+            break
+        elif guesses <= 0:
+            print(f'\033[7mYou lost. Guesses exceeded. The secret word was - {secret_word}\033[0m')
+            break
+        else:
+            print(get_guessed_word(secret_word, letters_guessed))
+            print(f'You have {guesses} guesses left.')
+            available = get_available_letters(letters_guessed)
+            print(f'Available letters: {available}')
+            while 1:
+                letter = input('Enter your guess letter:   ')
+                if letter.isalpha():
+                    break
+                else:
+                    if warnings <= 0:
+                        guesses -= 1
+                        if guesses == 0:
+                            break
+                    else:
+                        warnings -= 1
+                    print(
+                        f'\033[31mYou must enter a letter of the Latin alphabet. Not a number or symbol. {warnings} warnings left\033[0m')
```

Comment:
So - you managed to reduce a line on 2 characters? Was that a goal?

Reply:
When I press the combinations of "ctrl + alt + L" in my IDE it automaticlly tries to format code by PEP8. I think it tried to shorten the string as much as it can. If it makes code unreadable I can make in 1 string